### PR TITLE
Give RW permissions to WALE_TMPDIR for all

### PR DESCRIPTION
**Pull request by CyberDem0n**

### CODE REVIEW

File: postgres-appliance/configure_spilo.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def write_wale_command_environment(placeholders, overwrite, provider):
     else:
         return
     if not os.path.exists(placeholders['WALE_TMPDIR']):
-        os.makedirs(placeholders['WALE_TMPDIR'])
+        os.makedirs(placeholders['WALE_TMPDIR'], mode=0777)
```
Comment (avaczi) [inline postgres-appliance/configure_spilo.py]: Why is world readable/writable needed?

Comment (CyberDem0n) [inline postgres-appliance/configure_spilo.py]: configure_spilo.py is executed by root and therefore directory is writable only by root, but we want to give access there to the postgres user as well.

Actually I've also forgot about "sticky bit".

**Comment by avaczi** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**